saleor/discount/migrations/0049_migrate_sales_to_promotions.py

```diff
@@ -1,231 +1,10 @@
 # Generated by Django 3.2.18 on 2023-07-06 09:43
-from dataclasses import dataclass
-from typing import List
-
-import graphene
 from django.db import migrations
+from saleor.discount.tasks import SaleToPromotionConverter
```

Comment:
We shouldn't importing anything from the code, as we'll be not able to remove it. We should copy the function to the migration code